docker/planet/scripts/docker-entrypoint.sh

```diff
@@ -10,6 +10,17 @@ else
   PROTOCOL="http"
 fi
 
+if [ -z "$PARENT_PROTOCOL" ]
```

Comment:
And maybe we should write this as a function to combine with the above block and be a little more DRY